### PR TITLE
elasticsearch: Clear tool output before indexing

### PR DESCRIPTION
**Pull request by mistydemeo**
This is both huge, and difficult to normalize for Elasticsearch.

Fixes #7038

### CODE REVIEW

File: src/archivematicaCommon/lib/elasticSearchFunctions.py [hunk 1]
```diff
@@ -50,6 +50,35 @@
...
+    """
+    root = doc.getroot()
+    nsmap = { #TODO use XML namespaces from archivematicaXMLNameSpaces.py
+        'dc': 'http://purl.org/dc/terms/',
+        'm': 'http://www.loc.gov/METS/',
+        'p': 'info:lc/xmlns/premis-v2',
+        'f': 'http://hul.harvard.edu/ois/xml/ns/fits/fits_output',
+    }
+
+    #before_length = len(ElementTree.tostring(root))
```
Comment (Hwesta) [inline src/archivematicaCommon/lib/elasticSearchFunctions.py]: Remove this and after_length below?

File: src/archivematicaCommon/lib/elasticSearchFunctions.py [hunk 2]
```diff
@@ -50,6 +50,35 @@
 class ElasticsearchError(Exception):
     pass
 
+def remove_tool_output_from_mets(doc):
+    """
+    Given an ElementTree object, returns a copy of the document with
```
Comment (Hwesta) [inline src/archivematicaCommon/lib/elasticSearchFunctions.py]: Does this actually copy it, or return a modified ElementTree?

Comment (mistydemeo) [inline src/archivematicaCommon/lib/elasticSearchFunctions.py]: It actually modifies it in-place, now that you mention it, which is not what it implies right now. Not really worth doing a deep-copy for this, so I'll just document that it modifies it in-place (and remove the return).

**Comment by Hwesta** [comment]
:+1: Minor comments only.

File: src/archivematicaCommon/lib/elasticSearchFunctions.py (hunk 1 above)
Comment (mistydemeo) [inline src/archivematicaCommon/lib/elasticSearchFunctions.py]: Yeah, I'd kept them from the old version but I guess it's better just to kill them.

**Pull request merged**